tests/python/sdk/fixtures.py

```diff
@@ -56,3 +57,13 @@ def fxt_login(admin_user: str, restore_db_per_class):
     with client:
         client.login((user, USER_PASS))
         yield (client, user)
+
+
+@pytest.fixture
+def fxt_coco_dataset(tmp_path: Path, fxt_image_file: Path, fxt_coco_file: Path):
+    dataset_path = tmp_path / "coco_dataset.zip"
+    with ZipFile(str(dataset_path), "x") as f:
+        f.write(str(fxt_image_file), arcname="images/" + fxt_image_file.name)
+        f.write(str(fxt_coco_file), arcname="annotations/instances_default.json")
```

Comment:
```suggestion
    with ZipFile(dataset_path, "x") as f:
        f.write(fxt_image_file, arcname="images/" + fxt_image_file.name)
        f.write(fxt_coco_file, arcname="annotations/instances_default.json")
```

Pretty sure that `zipfile` supports path-like objects.

Reply:
Fixed